rag/multimedia_rag/video_rag.py

```
@@ -0,0 +1,400 @@
+        self._init_table()
+    
+    def _init_workspace(self):
+        """Initialize Pixeltable workspace"""
+        try:
+            pxt.drop_dir(self.workspace, force=True)
+            pxt.create_dir(self.workspace)
+        except Exception as e:
+            logger.warning(f"Workspace setup issue: {e}")
+            try:
+                pxt.get_dir(self.workspace)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)